fault-localization-toolset/src/main/java/se/de/hu_berlin/informatik/gen/ranking/modules/RankingModule.java

```diff
@@ -87,6 +87,9 @@ private void generateRanking(final ISpectra<T, ?> spectra,
             final Ranking<INode<T>> ranking = localizer.localize(spectra, strategy);
             Paths.get(outputdir + File.separator + subfolder).toFile().mkdirs();
             ranking.save(outputdir + File.separator + subfolder + File.separator + BugLoRDConstants.FILENAME_RANKING_FILE);
+            System.out.println("------- ranking saved -------"); //PT
+            System.out.println(ranking.getBestRankingElement()); //PT
+            System.out.println(ranking.getBestRankingValue()); //PT
```

Comment:
Wenn du schon Debug-Infos im Commit lässt, dann schreib wenigstens in der Ausgabe dazu, was du ausgibst. Schöner ist es wahrscheinlich auch, wenn du an der Stelle String.format(...) benutzt.